pitmutationmate/src/main/kotlin/com/amos/pitmutationmate/pitmutationmate/MutationTestToolWindowFactory.kt

```diff
@@ -24,4 +33,19 @@ internal class MutationTestToolWindowFactory : ToolWindowFactory, DumbAware {
         toolWindow.contentManager.addContent(lineChart)
         toolWindow.contentManager.addContent(barChart)
     }
+
+    fun updateReport(toolWindow: ToolWindow, newCoverageReport: XMLParser.CoverageReport) {
+        toolWindow.contentManager.getContent(0)?.component = LatestPiTestReport(newCoverageReport)
+    }
```

Comment:
Is there a way to maybe access the first content by name? Otherwise, if someone changes the order of content creation then this will break. If not I don't think this is a huge issue but if it's possible then it should probably be done.